internal/resources/policy/kind/network/recipe/constants.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package recipe
+
+const (
+	AllowAllKey          = "allow_all"
+	AllowAllToPodsKey    = "allow_all_to_pods"
+	AllowAllEgressKey    = "allow_all_egress"
+	DenyAllKey           = "deny_all"
+	DenyAllToPodsKey     = "deny_all_to_pods"
+	DenyAllEgressKey     = "deny_all_egress"
+	CustomEgressKey      = "custom_egress"
+	CustomIngressKey     = "custom_ingress"
+	FromOwnNamespaceKey  = "from_own_namespace"
+	toPodLabelsKey       = "to_pod_labels"
+	labelKey             = "key"
+	labelValueKey        = "value"
+	rulesKey             = "rules"
+	portsKey             = "ports"
+	portKey              = "port"
+	protocolKey          = "protocol"
+	ruleSpecKey          = "rule_spec"
+	ruleSpecIPKey        = "custom_ip"
+	ruleSpecSelectorKey  = "custom_selector"
+	ipBlockKey           = "ip_block"
+	cidrKey              = "cidr"
+	exceptKey            = "except"
+	namespaceSelectorKey = "namespace_selector"
+	podSelectorKey       = "pod_selector"
```

Comment:
nit: add a new line

Reply:
Till L32 the constants are defining keys and L33-L35 are not keys, so new line is to separate them for better readability

Reply:
Done